### ISSUE TITLE
does CentOS Stream affect the guides' CentOS OS recommendation?

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse#7470]
A placeholder issue to track documentation changes pending tech hours discussion per @pdurbin 

CentOS 8, formerly supported until the EOL of RHEL 8 (2029), now EOLs in 2021.

CentOS-Stream-8 is an easy conversion from CentOS 8 but follows a "rolling release" model which may concern administrators of production installations. It's EOL is currently listed as "N/A" with an expected CentOS-Stream-9 beta appearing in Q2 2021.

Personal note: I converted a test system painlessly and in under 5 minutes, though there was a bit of dependency tomfoolery with openblas-devel (related to R). R's package dependencies is often a source of noise during upgrades, so this shouldn't be a surprise.

The bottom line seems to stress an organizational decision on whether to pay for RHEL support or depend on free software that will stay just ahead of the RHEL release, and how we represent that in the installation guide.

[CentOS Stream FAQ](https://www.redhat.com/en/blog/faq-centos-stream-updates) FWIW

**Comment by donsizemore** [comment]
The current replacement front-runner / darling appears to be RockyLinux, which estimates an initial release in Q2 2021: https://www.zdnet.com/article/rocky-linux-first-release-is-coming-in-q2-2021-say-developers/

**Comment by poikilotherm** [comment]
Shouldn't we also recommend people to take a look if they are eligible for the free RHEL license?

**Comment by donsizemore** [comment]
[Rocky Linux](https://rockylinux.org/) announces: `The current ETA for initial release is March 31.`

**Comment by pdurbin** [comment]
See also the "CentOS 8 alternative for Prod server" thread at https://groups.google.com/g/dataverse-community/c/tOzZx8OzyQ8/m/N4SaMj9EBAAJ especially:

> "as you may already know that the CentOS 8 will have no LTS after December 2021. that makes it a very wild decision to deploy a production server on such OS although it is recommended by the original documentation."

**Comment by donsizemore** [comment]
AlmaLinux 8.3 was released on March 30th, and offers a CentOS-to-Alma migration script: https://almalinux.org/blog/almalinux-os-stable-release-is-live/

RockyLinux has pushed its beta release to April 30th.
